### PR TITLE
Pull request template for schema changes

### PR DESCRIPTION
**Pull request by vcschapp**
# Description

This change adds a pull request template for schema changes. Once merged, all new schema change PRs should automatically receive the template, and follow it.

# Testing

N/A. Not possible to test without pushing the change first. However, follows [the pattern from *tf-transportation*](https://github.com/OvertureMaps/tf-transportation/blob/bf9dfb218aec050a85c585487a1163765d63d2ff/.github/pull_request_template.md), which works.

### CODE REVIEW

**Review by jenningsanderson — APPROVED** [review]
🎉

**Comment by jenningsanderson** [comment]
It drives me crazy we can't get the PR # to autofill... I played around with that and couldn't figure it out, so this works!

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+1. [ ] Add relevant examples.
+2. [ ] Add relevant counterexamples.
+3. [ ] Update in-schema documentation using plain English written in complete sentences, if an update is required.
+4. [ ] Update Docusaurus documentation, if an update is required.
+5. [ ] Review change with Overture technical writer to ensure any advanced documentation needs will be taken care of, unless the change is trivial and would not affect the documentation.
+
+# Documentation Website
+
+*Update the hyperlink below to put the pull request number in.*
+
```
Comment (DavidKarlas) [inline .github/pull_request_template.md:32]: Should we explain this can only be edited after PR is created?

**Review by DavidKarlas — APPROVED** [review]

**Comment by DavidKarlas** [comment]
> It drives me crazy we can't get the PR # to autofill... I played around with that and couldn't figure it out, so this works!

I guess we could have a bot that just does this(posts new comment with link).

**Review by TristanDiet-TomTom — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (vcschapp) [reply .github/pull_request_template.md:32]: See how it plans out, and might be worth an effort in an update?

**Pull request merged**